R/setkey.R

```diff
@@ -86,12 +86,26 @@ setkeyv <- function(x, cols, verbose=getOption("datatable.verbose"), physical=TR
     if (!typeof(.xi) %chin% c("integer","logical","character","double")) stop("Column '",i,"' is type '",typeof(.xi),"' which is not supported as a key column type, currently.")
   }
   if (!is.character(cols) || length(cols)<1L) stop("Internal error. 'cols' should be character at this point in setkey; please report.") # nocov
-  if (verbose) {
-    tt = suppressMessages(system.time(o <- forderv(x, cols, sort=TRUE, retGrp=FALSE)))  # system.time does a gc, so we don't want this always on, until refcnt is on by default in R
-    # suppress needed for tests 644 and 645 in verbose mode
-    cat("forder took", tt["user.self"]+tt["sys.self"], "sec\n")
+  
+  # get existing index name if any
+  found_index <- NULL
```

Comment:
We prefer `=` in data.table.  I don't agree with the common and widely marketed advice to use `<-`.  Single `=` is the same as C and many other languages.  I sometimes hear, e.g. from Python folk,  that R code using `<-` looks "old"/"not modern" and I see their point.  I've always used `=`.   I save `<-` for use when passing function arguments to do an assign and pass in one go;   e.g.   `write(DT, file=f<-tempfile());   ... do something with f ...` .   I'm often swapping between C and R (which I think more people should do too since C is not as hard as some people want you to believe).  When doing this, using `=` to mean assign and `==` to mean equals in two languages consistently (R and C) is nice.   And by the way,  the people who laugh at R for using `L` postfix to mean integer ... it comes from C (it's the same as C) and it makes a lot of sense.

Reply:
thanks for detailed insight. made the change.